src/pal/src/thread/process.cpp

```diff
@@ -1468,6 +1471,9 @@ class PAL_RuntimeStartupHelper
     // Debugger waits on this semaphore and the runtime signals it on startup.
     sem_t *m_startupSem;
 
+    // Debuggee waits on this semaphore and the debugger signals it after the callback returns.
```

Comment:
Its probably not apparent in context here what 'the callback' refers to. Perhaps just say debugger signals this when it  is ready for the debuggee to resume executing?